lin_vel_array:        [1.0,0.5,0.25]
ang_vel_array:        [1.0,3.0,2.0]
duration_array:       [10.0,5.0,10.0]
